pkg/sql/schemachanger/sctest/framework.go

```diff
@@ -880,9 +875,9 @@ func executeSchemaChangeTxn(
 					err = tx.Commit()
 				}
 			}()
-			for _, stmt := range spec.Stmts {
+			for i, stmt := range spec.Stmts {
 				if _, err := tx.Exec(stmt.SQL); err != nil {
-					return err
+					return errors.Wrapf(err, "error processing statement %d: %s", i, stmt.SQL)
```

Comment:
nit: is something like ("error processing %d-th statement %q", i, stmt.SQL) better?

Reply:
I changed it to `statement[%d]` and output the statement with `%q` instead of `%s` to match similar messages. Using "-th" only works for some numbers and is confusing for things like 1-th, 2-th, etc.